code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -77,8 +77,12 @@
 		var/mob/living/L = holder.my_atom
 		if(L.stat != DEAD)
 			e.amount *= 0.5
-	e.start()
-	holder.clear_reagents()
+	e.start()    
+	if(ismob(holder))
```

Comment:
Holy mother of snowflake batman.

If what you want to do is remove the reagent clearing (which I see no evidence of being a bug, for the record) then what is this hardcoded shit doing here?
The chemical reaction already implicitly removes potassium and water.